### PR TITLE
Include bagit version in Bag-Software-Agent

### LINKED ISSUE #41: Bag-Software-Agent include version?
It would be nice if the default Bag-Software-Agent had a version in it. Currently it looks like:

    bagit.py <http://github.com/libraryofcongress/bagit-python>

But maybe something like this would be better, for discovering bags created with (gasp) a version of bagit.py that has a bug?

    bagit.py v1.5.3 <http://github.com/libraryofcongress/bagit-python>

### PR DESCRIPTION
**Pull request by Hwesta**
Fixes #41

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -24,10 +24,12 @@
 except:
     requirements.append("hashlib")
 
+import bagit
```
Comment (Hwesta) [inline setup.py]: Moved and rebased.

Comment (johnscancella) [inline setup.py]: I believe the PEP-8 convention is to have all the "plain" imports at the top followed by the "from X import Y" declarations

Comment (Hwesta) [inline setup.py]: Hmm, I was following [the section](https://www.python.org/dev/peps/pep-0008/#imports) "Imports should be grouped in the following order: 1. standard library imports 2. related third party imports 3. local application/library specific imports" and didn't see anything about `import x` coming before `from x import y`. Happy to move it all the way to the top if you want.

Comment (johnscancella) [inline setup.py]: ahh ok. I don't do as much python as other languages so I probably am miss-remembering. Doesn't matter too much to me. I would say with your first merge that it looks good to me.

**Comment by Hwesta** [comment]
This discussion is late now, I discovered but https://packaging.python.org/single_source_version/ which has advice on storing the version in only one place.

**Comment by johnscancella** [comment]
@Hwesta great find! If you want to continue on this and submit another pull request that would be great.

**Pull request merged**